Getting-Started-with-Akka-http-signature.md

```diff
@@ -270,7 +270,7 @@ $ ln -s pubKey.ttl pubKey
 
 Currently in order to manipulate the access control one can use `curl` and `PATCH` commands [As detailed in the rww-play wiki page](https://github.com/read-write-web/rww-play/wiki/Curl-Interactions).
 
-To demonstrate how access control changes user permitted actions, one can also manually edit the .acl.ttl file. To do this, one must navigate to the test_www directory (or the server that contains the Key files) and look for the correct file. After that in order to permit access to the , the user must comment out the following:
+To demonstrate how access control changes user permitted actions, one can also manually edit the .acl.ttl file. To do this, one must navigate to the test_www directory (or the server that contains the Key files) and look for the correct file. After that in order to permit access to the , the user must comment out the `acl:agentClass <http://xmlns.com/foaf/0.1/Agent> ;`. This expression gives everyone permission to access this file and by commenting it out the user makes sure access is restricted only to them.
```

Comment:
"access to the , " ???
"access is restricted only to them" <- this means you give permission to everyone since "them" refers to the last  thing you referred to in the sentence.  People will guess what is meant, but documentation is not meant to be unecessarily require guessing.